src/components/ShowMoreScholarships.jsx

```diff
@@ -0,0 +1,32 @@
+import { useContext } from 'react';
+import ScholarshipsContext from '../models/ScholarshipsContext';
+import ShowMoreScholarshipCard from './ShowMoreScholarshipCard';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+
+export default function ShowMoreScholarships() {
+  const { canLoadMore, error, loading, loadMore, scholarships, setFilters } =
```

Comment:
canLoadMore, loadMore & setFilters are not necessary since users are only going to get a subset of scholarships.